modules/standard/BitOps.chpl

```diff
@@ -157,7 +157,7 @@ module BitOps {
  * module to hide the extern procedures
  */
 pragma "no doc"
```

Comment:
My concern with this patch is just getting buy-in on "has private been applied correctly here/should it be applied somewhere else as well"